### PR TITLE
Ore priority fix

### PR DESCRIPTION
**Pull request by Slava0135**
Sand low priority is hard coded. Bad for mods having floors with drops.

~~For some reason other.floor().playerUnmineable() is always true so priority doesn't work~~

### CODE REVIEW

**Comment by Slava0135** [comment]
Now it's working. The reason I am complaining about it is cos in my mod I made snow floors drop snowballs but the thing is that it makes hard to mine ores on snow...

**Comment by Slava0135** [comment]
with fix:

now

File: core/src/mindustry/world/blocks/production/Drill.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import mindustry.world.meta.*;
 import mindustry.world.meta.values.*;
 
+import java.util.Comparator;
```
Comment (summetdev) [inline core/src/mindustry/world/blocks/production/Drill.java:23]: `java.util` import detected :eyes:

> Do not use incompatible Java features.

**Review by summetdev — CHANGES_REQUESTED** [review]

File: core/src/mindustry/world/blocks/production/Drill.java (hunk 1 above)
Comment (Slava0135) [reply core/src/mindustry/world/blocks/production/Drill.java:23]: oh no

**Comment by Slava0135** [comment]
no fix
before

**Comment by BlueWolf3682** [comment]
amazing item sprite, 11/10 reskin

**Comment by Slava0135** [comment]
> amazing item sprite, 11/10 reskin

~~is that a white coal or coke?~~

**Comment by BlueWolf3682** [comment]
coal's evil twin brother

File: core/src/mindustry/world/blocks/production/Drill.java [hunk 2]
```diff
@@ -157,9 +159,15 @@ void countOre(Tile tile){
         oreCount.clear();
         itemArray.clear();
 
+
```
Comment (summetdev) [inline core/src/mindustry/world/blocks/production/Drill.java:162]: two empty lines.

```suggestion

```

File: core/src/mindustry/world/blocks/production/Drill.java (hunk 1 above)
Comment (summetdev) [reply core/src/mindustry/world/blocks/production/Drill.java:23]: (you should remove `import` too)

**Comment by Slava0135** [comment]
a N u K e D i D i T h I m S e L f

**Pull request closed without merging**